### ISSUE TITLE
Unable to upload document

### ISSUE THREAD

**Issue by jocull** [Dwolla/dwolla-v2-node#4]
I'm following the example for using `FormData` exactly, but getting an error.

It seems like the `post` method is not detecting that the variable is an instance of `FormData`? Is there a special trick to making this work? Are you supposed to `npm install form-data` locally?

```
Error: the object {
  "body": {
    "code": "UnsupportedRequestContentType"
    "message": "Invalid request Content-Type. Expected 'multipart/form-data'."
  }
  "headers": {
    "_headers": {
      "access-control-allow-origin": [
        "*"
      ]
      "cf-ray": [
        "2b281143d6890166-ORD"
      ]
      "connection": [
        "close"
      ]
      "content-length": [
        "114"
      ]
      "content-type": [
        "application/vnd.dwolla.v1.hal+json; profile=\"http://nocarrier.co.uk/profiles/vnd.error/\"; charset=UTF-8"
      ]
      "date": [
        "Mon, 13 Jun 2016 19:49:44 GMT"
      ]
      "server": [
        "cloudflare-nginx"
      ]
      "set-cookie": [
        "__cfduid=d5704accff6f82667d9f9f856c918b68f1465847383; expires=Tue, 13-Jun-17 19:49:43 GMT; path=/; domain=.dwolla.com; HttpOnly"
      ]
    }
  }
  "status": 415
} was thrown, throw an Error :)
```

**Comment by jocull** [comment]
Not sure if this was a `npm dedupe` problem or something else, but I reinstalled `form-data` and now everything seems fine...

**Comment by sausman** [comment]
Reopening (someone else had the same issue).

**Comment by jocull** [comment]
@sausman If it helps, I think the problem arises if you `npm dedupe`. You need to install your own dependency on `form-data` and not attempt to use the one nested in the `dwolla-v2` package.

**Comment by sausman** [comment]
Thanks @jocull, just released `1.1.2` with the fix.
